recipes/fdint/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "fdint" %}
+{% set version = "2.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30db139684d362652670e2cd3206b5dd7b3b93b86c3aff37f4b4fd4a3f98aead
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - fdint
+    - fdint.examples
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://scott-maddox.github.io/fdint
```

Comment:
This site does not seem to exist

Reply:
Good catch! I had already fixed this, but it must have gotten lost in the rebase/force pushing...